src/server/routes/api.ts

```diff
@@ -1121,6 +1145,23 @@ connection.once('open', () => {
             }
         };
     };
+
+    const getGeneCorrelationData = (hgncId: string) => {
+        if (!genesNeuroCorr.length) {  // if DB has error
+            console.log('getGeneCorrelationData: Mongo DB return document length ', genesNeuroCorr.length);
+            return [];
+        }
+
+        const ensembledIds = hgncToEnsembl.get(hgncId);
+        if (ensembledIds !== undefined) {
```

Comment:
if (ensembledIds === undefined)  is there an error? Should we output something?

 
           
        }

Reply:
No, but I did forget to return a value if the ensembleIDs is undefined but I have just fixed it now. Thanks for catching that!